### PR TITLE
Remove ACL2_SAVE_QUICKLISP hack in Makefile

### PR DESCRIPTION
**Pull request by kini**
The comment says that if this hack is no longer needed by December
2015, it should be removed.  Clearly, it is already past December
2015, so we should remove it.

---

Or so confidently declares my commit message, but what do you think?

### CODE REVIEW

**Comment by ragerdl** [comment]
Thanks for the `clean`up.  Did you test this with a fresh checkout?  I agree that based on my comments that it should be reasonable to remove.  If not, maybe I'll wipe out the leeroy workspace before it gets sent over to leeroy.

**Comment by kini** [comment]
I didn't test it, no -- sorry about that. I can test it if you want to avoid slowing down leeroy by clearing the workspace. (Though probably it's good to clear out the workspace from time to time.)

**Comment by ragerdl** [comment]
Alright, I've had time to think about this now.  It seems to me that the dependency indicated by `clean: quicklisp_clean` needs to remain.  Can you put back that one line and then update this pull request stating as such?  It should be ready to merge at that point.

File: books/GNUmakefile [hunk 1]
```diff
@@ -529,25 +529,6 @@ quicklisp_clean:
...
-# quicklisp/inst/cache/asdf-installs directory), and so he prefers
-# that the "hack" be preserved.
-
-# May 2015 update: since the quicklisp installation procedure has
-# changed, the conditional part of this code is hypothetically dead.
-# We leave it commented for now, but if December 2015 comes around,
-# and it's still unnecessary, the conditional part of this (and the
-# above comments) should be deleted.
-#ifndef ACL2_SAVE_QUICKLISP
-clean: quicklisp_clean
```
Comment (ragerdl) [inline books/GNUmakefile]: Please put back this dependency

Comment (kini) [reply books/GNUmakefile]: Done.

**Comment by ragerdl** [comment]
Thanks for the cleanup, I went ahead and "Squashed and merged" your pull request.

**Pull request merged**